src/components/Calculator.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
While you did an amazing job working on the project requirements I noticed you disabled eslint in this given file. It is against microverse policy to disable linters because linters help us write clean code it is also not good practice to disable linters so it would be nice if you enabled eslint in this file. Please kindly consider doing this in all files were eslint was disabled.